cylc/flow/daemonize.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
@sgaist I think you created this branch, off your other branch? It may be better to create both from `master`, or wait until your first PR gets reviewed... or you can wait until the other PR gets review/merged, then rebase this one :+1: 

The `setup.py` will need a small change in the other branch, which means this one will probably be in conflict...

Reply:
You're thinking is correct.

Yes, that's was the idea: get this one reviewed in parallel while the first is updated/integrated, then do a rebase here. I like to have history straightforward :-)

Reply:
Righto, but that doesn't pass

>Does not contain off-topic changes (use other PRs for other changes).

If you create this PR from master, then we can merge the first PR that passes Travis tests. But right now this PR has a dependency on the other PR. Reviewers could also review something here by accident, without seeing the other PR.

@sgaist could you separate and have just the license and copyright fixes on this one, please?

Reply:
Sure thing, sorry I pushed a new version already, I didn't saw your message. I'll make that one from master too then.

Reply:
Our test battery may take a long time, and I would like to see this PR blocked while we wait for the other one (just in case anything goes wrong there). Hopefully both will be merged soon :crossed_fingers: ! Thanks a lot!

Reply:
Done !

Reply:
That was quick! I will give Travis some time now to check the changes and take another look before logging off :+1: (I'm on NZ time, but others are based in different TZs, so there may be other devs reviewing this one too soon)